spotify-widget/spotify.lua

```diff
@@ -33,7 +34,7 @@ local function worker(user_args)
 
     local play_icon = args.play_icon or '/usr/share/icons/Arc/actions/24/player_play.png'
     local pause_icon = args.pause_icon or '/usr/share/icons/Arc/actions/24/player_pause.png'
-    local font = args.font or 'Play 9'
+    local font = args.font or beautiful.font
```

Comment:
Following my comment, here is what I think should be the reasonable way to do :

```suggestion
    local font = args.font or beautiful.font or "Play 9"
```